### PR TITLE
Inline recirc

### PR DESCRIPTION
**Pull request by MichaelButkovic**

### CODE REVIEW

File: bulbs/content/filters.py [hunk 1]
```diff
@@ -36,6 +36,18 @@ def parse_datetime(value):
         raise ValueError('Value must be parsable to datetime object. Got `{}`'.format(type(value)))
 
 
+def _parse_slugs(slugs):
```
Comment (mparent61) [inline bulbs/content/filters.py]: Nice!

File: bulbs/recirc/mixins.py [hunk 2]
```diff
@@ -64,6 +65,13 @@ def get_full_recirc_content(self, published=True):
         })
         return search
 
+    def get_inline_recirc_content(self, published=True):
+        qs = Content.search_objects.search()
+        qs = qs.query(FeatureTypeBoost(slugs=["video"]))
+        qs = qs.query(TagBoost(slugs=self.tags.values_list("slug", flat=True)))
+
+        return qs[:3]
```
Comment (mparent61) [inline bulbs/recirc/mixins.py]: should be a def parameter, `count=3`

File: tests/recirc/test_recirc_views.py [hunk 3]
```diff
@@ -111,3 +111,33 @@ def test_recirc_unpublished(self):
...
+            published=timezone.now() - timezone.timedelta(days=1)
+        )
+        content.tags.add(tag)
+
+        # check that they are returned in the response
+        recirc_url = reverse('content_inline_recirc', kwargs={'pk': content.id})
+        response = self.api_client.get(recirc_url)
+        self.assertEqual(response.status_code, 200)
+
+        import pdb; pdb.set_trace()
```
Comment (mparent61) [inline tests/recirc/test_recirc_views.py]: AHHH!

**Comment by mparent61** [comment]
Ah, I see still WIP :)

**Comment by MichaelButkovic** [comment]
Yeah, I'm getting duplicate articles in the queryset, so I'm still debugging that. Figured it'd still be worthwhile to open it up now though.

**Comment by benghaziboy** [comment]
fix tests. LGTM

**Comment by MichaelButkovic** [comment]
@mparent61 ready to go now, if you want to take a look too

**Comment by mparent61** [comment]
:+1:

**Pull request merged**